README.md

```diff
@@ -356,5 +356,114 @@ function `bodn2c`, and wants the vector version if it exists (but it doesn't!),
+no active elements.
+
+If you already have data that you want to convert to a spice cell, you can use a simpler
+interface:
+```python
+spice_cell = SpiceCell([1, 2, 3, 4, 5])
+spice_cell = SpiceCell(np.arange(1.0, 10.0))
+spice_cell = SpiceCell(np.arange(1.0, 10.0), size=40)  # set size explicitly
+```
+If you use these constructions, Python will create a `SpiceCell` whose size is a little 
+bigger than the length of the given data. The passed first argument will be the active
+elements.
+
+Any CSPICE function that expects a `SpiceCell` as an input can also be passed an array,
+a tuple, or anything that can reasonably be converted into an appropriately typed
+`SpiceCell`:
+
+```shell
+>>> import cspyce
+>>> cspyce.wninsd(2.0, 4.9, (1.0, 3.0, 5.0, 7.0, 9.0, 11.0))
+<SpiceCell float 6/12 [ 1.   4.9  5.   7.   9.  11. ]>
+```
+
+#### Sequence-like operations
+
+`SpiceCell` offers the following methods so that they behave like Python sequences.
+
+`spice_cell[index]`, `spice_cell[index] = value`
+
+:  Get or sets the `index`-th element of the `SpiceCell`. 
+   Must have `-len(spice_cell) ≤ index < spice_cell.size`.  As in Python, we allow
+   negative indices, but -1, -2, etc count backwards from the last active elements.
+
+`len(spice_cell)`
+: Returns the number of active elements of the `SpiceCell`.
+
+`spice_cell.append(value)`
+: Appends a new active cell to the end of the `SpiceCell`'s active cells.
+The cell's maximum size is grown if necessary. 
+
+`spice_cell.extend(values)`
+: Appends all the values of the sequence to the end of the `SpiceCell`'s active cells.
+The `SpiceCell`'s maximum size is grown if necessary.
+
+`spice_cell += values`
+: Synonym for `spice_cell.extend(values)`
+
+`spice_cell.clear()`
+: Make all elements inactive. Same as `spice_cell.card = 0`. 
+
+`iter(spice_cell)` 
+: Iterates through the active elements of `SpiceCell`.
+Because of this iterator, `list(spice_cell)`, `set(spice_cell)`, `tuple(spice_cell)` 
+work as expected.
+You can also write a for loop `for item in spice_cell: ...` to iterate through
+the active elements.
+
+#### SpiceCell specific operations.
+
+`SpiceCell` also has the following methods and properties specific to a `SpiceCell`.
+
+`spice_cell.size`
+: Gets the current maximum size of the `SpiceCell`.
+
+`spice_cell.size = value`
+: Grows or shrinks the maximum size of the `SpiceCell`.
+If the number of active elements is greater than `value`, it is reduced to `value`.
+This operation is a `cspyce` enhancement does not exist in CSPICE.
+
+`spice_cell.card`
+: Same as `len(spice_cell)`.  (Also known as the "cardinality". Hence the name).
+
+`spice_cell.card = value`
+: Changes the number of active elements in the `SpiceCell`. 
+Must have `0 ≤ value < spice_cell.size`.
+
+`spice_cell.as_array()` 
+: Shows the active contents of the array as a numpy array
+
+`spice_cell.as_intervals` 
+: Shows the active contents of the array as a 2-dimensional `card/2`x 2 array.  
```

Comment:
```suggestion
`spice_cell.as_intervals()` 
: Returns the active contents of the array as a 2-dimensional `card/2`x 2 array.  
```

Reply:
Mark wanted the word "view" in there, so I changed it to "returns a view of...."